Tests/MapboxMapsTests/Snapshot/MapboxMapsSnapshotTests.swift

```diff
@@ -1,10 +1,134 @@
 import XCTest
+import MapboxMaps
+import MapboxCoreMaps
+import Turf
+import Foundation
+import UIKit
 #if canImport(MapboxMaps)
 @testable import MapboxMaps
 #else
 @testable import MapboxMapsSnapshot
 #endif
-
 //swiftlint:disable explicit_top_level_acl explicit_acl
-class MapboxMapsSnapshotTests: XCTestCase {
+class MapboxMapsSnapshotTests: IntegrationTestCase {
+   
+    var geoJSONSource = GeoJSONSource()
+    var snapshotView = UIImageView()
+    var blankAttachment: XCTAttachment!
+    var newAttachment: XCTAttachment!
+    
+    // Create snapshot options
+    private func snapshotterOptions(size: CGSize) -> MapSnapshotOptions {
+        let resourceOptions = ResourceOptions(accessToken: accessToken)
```

Comment:
We recently added an extension on XCTestCase that can retrieve the access token for you. This may allow you to remove the inheritance from IntegrationTestCase